src/sentry/integrations/pagerduty/utils.py

```diff
@@ -131,7 +132,7 @@ def attach_custom_severity(
         return data
 
     severity = app_config.get("priority", None)
-    if severity is not None:
+    if severity is not None and severity in PAGERDUTY_DEFAULT_SEVERITY:
```

Comment:
`PAGERDUTY_DEFAULT_SEVERITY` is a string?

```suggestion
    if severity is not None and severity in PagerdutySeverity:
```

Reply:
whoops it needs to be severity == `PAGERDUTY_DEFAULT_SEVERITY`